english/madness/yogsothoth/yog-sothoth.md

```diff
@@ -1,4 +1,7 @@
-Mercifully, the Yog-Sothoth uses only a tiny fraction of his epic power 
+Mercifully, the Yog-Sothoth uses only a tiny fraction of his epic power
 and sweet death descends upon you.
 
+A voice booms!
+"Yog-Sothoth knows the gate. Yog-Sothoth is the gate. Yog-Sothoth is the key and guardian of the gate. Past, present, future, all are one in Yog-Sothoth. He knows where the Old Ones broke through of old, and where They shall break through again. He knows where They have trod earth's fields, and where They still tread them, and why no one can behold Them as They tread."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.